client/src/components/OrganizerDashboard.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const OrgnizerDashboard = () => (
+  <div> OrgnizerDashboard </div>
```

Comment:
Lots going on here in the `OrgnizerDashboard`. 

I personally prefer the `OrgnizerDashboard` spelling over `OrganizerDashboard`